h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -2968,6 +2968,7 @@ def compare_frames(frame1, frame2, numElements, tol_time=0, tol_numeric=0, stric
 
```

Comment:
Ideally I would merge the compare_frames and compare_frames_local into one function, but I can see that would be big task

Reply:
Yeah and I am lazy.

Reply:
what about making compare_frames_local call compare_frames? that would be good middle of the road solution